admin/templates.md

```diff
@@ -14,56 +12,70 @@ located in Git repositories.
 
 ![Enable workspace templates](../assets/enable-ws-templates.png)
 
-## Template policy (alpha)
+## Template policy
+
+> Template policies are currently an **alpha** feature.
 
 If you enable the use of workspace templates, a **template policy** allows you
-to control which fields users can set and which values can be used when they
-define their workspaces.
+to control which fields users can set and which values can they can use when
+defining their workspaces.
+
+Users can apply the following policies to fields:
+
+- `read` - **Default.** Workspaces cannot modify the field
+- `write` - Workspaces can overwrite the field
+- `append` - Workspaces can append lists (e.g., configure.start steps) or
```

Comment:
is the comma here intended? @khorne3 `e.g,`?

Reply:
@bpmct yup it is!